docs/DeveloperGuide.md

```diff
@@ -1199,90 +1216,215 @@ Expected: The most recent window size and location is retained.
-**Current Implementation**: In the current version of our application,  when a new internship application is added to our list, it is placed at the bottom. This positioning often renders the new entry outside the user's current view, requiring them to manually scroll down to locate the latest addition.
 
-**Planned Enhancement**: We are planning to enhance the user experience by changing how newly added internship applications are displayed. The key improvement will be automatically scrolling the list to bring the newly added application into view immediately. Additionally, we are considering placing new entries at the top of the list rather than at the bottom.
+**Current Implementation**: In the current version of our application, when a
+new internship application is added to our list, it is placed at the bottom.
+This positioning often renders the new entry outside the user's current view,
+requiring them to manually scroll down to locate the latest addition.
+
+**Planned Enhancement**: We are planning to enhance the user experience by
+changing how newly added internship applications are displayed. The key
+improvement will be automatically scrolling the list to bring the newly added
+application into view immediately. Additionally, we are considering placing new
+entries at the top of the list rather than at the bottom.
 
 **Proposed Changes**:
 
-* **Automatic Scrolling**: Modify the list view functionality to automatically scroll to the position of the newly added internship application. This ensures that the latest entry is immediately visible to the user without requiring manual navigation.
+* **Automatic Scrolling**: Modify the list view functionality to automatically
+  scroll to the position of the newly added internship application. This ensures
+  that the latest entry is immediately visible to the user without requiring
+  manual navigation.
+
+* **List Ordering Modification**: Change the default order of the list to
+  display new applications at the top. This approach aligns with common user
+  expectations, where the most recent items are readily accessible.
+
+**Expected Benefits**:
+
+* **Improved Visibility of New Applications**: By immediately showcasing new
+  applications, users can immediately see and interact with the most recent
+  entries, improving workflow efficiency.
+* **Adaptation to User Preferences**: Adjusting the list order to display new
+  items at the top aligns with common user behavior and expectations, thereby
+  improving usability.
+* **Enhanced User Experience**: Automatically scrolling to new entries saves
+  time and improves the overall user experience, making the application more
+  intuitive and efficient.
+
+### Sorting by Multiple Fields
+
+**Current Implementation**: Currently, the application allows sorting of data by
+a single field at a time using commands such as `c/a` for sorting cycles in
+ascending order or `n/d` for sorting company names in descending order.
```

Comment:
```suggestion
**Current Implementation**: Currently, the application only allows sorting of internship applications by
ONE field at a time using parameters such as `c/a` for sorting by cycles in
ascending order or `n/d` for sorting by company names in descending order.
```

Perhaps this would be a better way to be both specific and clear with the current implementation of the `sort` command?

Reply:
Also, I suggested the change to "Currently" to follow the other planned enhancements!